runelite-client/src/main/java/net/runelite/client/plugins/farmingtracker/FarmingTrackerConfig.java

```diff
@@ -48,6 +48,16 @@ default boolean estimateRelative()
 		return false;
 	}
 
+	@ConfigItem(
```

Comment:
Don't make this toggle

Reply:
Agreed. The client doesn't really support non-English languages at the moment, and using the player's locale for dates will sometimes display them in foreign languages.